### DISCUSSION TITLE
replace /command with simple menu button

### DISCUSSION THREAD

**Discussion by skynight137** [anasty17/mirror-leech-telegram-bot#920] [Ideas]
hi brother,
can u replace /cmd with menu button???

**Comment by anasty17** [comment]
No, i don't wamt to press button for each link/file. Direct cmd easier for me. And multi link will not be good enough and can't use rss

**Reply by skynight137** [reply]
ok, emb,  how to use rss?? can u give example using rss!!
is the db correct?, I used heroku var to set it, I think the /addsudo command entered the db correctly, but the message "Error in DB connection, check log for details" appears??  is there something wrong??

**Comment by anasty17** [comment]
Read readme and /help ! Addsudo not related to database! READ README

**Reply by skynight137** [reply]
i have removed DATABASE_URL -> #DATABASE_HEROKU in config.env!, and enabled it on heroku postgres?  is it wrong?

**Reply by anasty17** [reply]
No not wrong but u should remove the var completely from config.env. not to leave it empty

**Reply by anasty17** [reply]
Send logs

**Reply by skynight137** [reply]
omg, it's too late and I'm feeling tired. Sorry, I forgot to reset the bot 😂😂. yeah, this is perfect dl bot, I'll keep waiting for the next update 👍👍

**Comment by anasty17** [comment]
Ur problem in database url! Since u didn't read the readme well!
